DependencyInjection/DoctrineExtension.php

```diff
@@ -343,6 +344,11 @@ protected function ormLoad(array $config, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('orm.xml');
 
+        // This check can be removed when support for Symfony < 4.2 is dropped
+        if (!method_exists(AbstractController::class, 'addLink')) {
```

Comment:
This checks the version of the FrameworkBundle which doesn't have to be the same as the one of the DoctrineBridge (which provides the type).

Reply:
So we also need to check whether the type implements the ResetInterface?